SpotifySearch/SpotifySearch/CellController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  CellController.swift
+//  SpotifySearch
+//
+//  Created by Marty Avedon on 10/28/16.
+//  Copyright © 2016 Marty Hernandez Avedon. All rights reserved.
+//
+
+import UIKit
+
+class CellController: UITableViewController {
+
+    var ourUnformattedURL: String = "https://api.spotify.com/v1/search?q=swift&type=album&limit=50"
+    var albumArr: [Albums]? = [Albums]()
+    var chosenAlbum: Albums?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.title = "Albums"//we need to alter this to represent the search term from the URL in something resembling English...we could be a punk and hardcode it to Blue but that would not be sporting
+        
+        AlbumFactory.RCA.getData {(data: Data?) in
+            dump(self.albumArr)
+        }
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return albumArr!.count //problems binding [Albums]...so we're forcing it. Bad.
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "eachAlbum", for: indexPath)
+        
+        // Adding content to cell
+        
+        let thisCellsAlbum = self.albumArr?[indexPath.row]
+        
+        // Configure the cell...
+        
+        cell.textLabel?.text = /*"ASDsccsdcw"*/ thisCellsAlbum?.title // this calls up the main text label -- in this case, the one named title in the storyboard. we may think we're "creating" the label but it's already waiting for us. this is more clear when we use a basic or detail cell style. we just don't have to create an outlet for it using control-drag, which is a little confusing
```

Comment:
You can use optional binding and conditionals if you're unsure

`
if let text = thisCellsAlbum.title {
 cell.textLabel?.text = text
} else {
 cell.textLabel?.text = "Not working"
}
`